src/main/webapp/selectScript.js

```diff
@@ -11,25 +11,20 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
-window.addEventListener('load', () => {
+export const CollegePlanner = (function() {
   let selected = [];  // Courses selected by the user
   let courses = [];   // List with all courses
 
-  document.getElementById('add-selected').addEventListener('click', () => {
-    addToSelected();
-  });
-
   /**
    * Gets courses from /courselist servlet to populate dropdown list
    */
   async function getOptions() {
     const response = await fetch('/courses');
-    courseList = await response.json();
+    const courseList = await response.json();
```

Comment:
hm, i feel like linters should have caught this. wonder if we need to fix something up later.